internal/checker/terraform/rules/ibm/app_id_management/tfibm_appid_mfa_sms_config_configured.rego

```diff
@@ -0,0 +1,63 @@
+# METADATA
+# title: "Verify MFA channel sms is properly configured"
+# description: "This policy verifies that the 'sms_config' block and its attributes 'key', 'secret', and 'from' exist in the IBM AppID MFA channel resource. It ensures that the SMS configuration for Multi-Factor Authentication (MFA) is properly set, enhancing the security of the application."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/appid_idp_mfa_channel
```

Comment:
Related Resources Link: `Page not Found`
[here](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/appid_idp_mfa_channel)